mdal/frmts/mdal_ugrid.cpp

```diff
@@ -525,6 +527,27 @@ void MDAL::DriverUgrid::parseNetCDFVariableMetadata( int varid,
         *isX = false;
         name = MDAL::replace( standardName, "_y_", "" );
       }
+      else if ( MDAL::contains( standardName, "_speed" ) )
+      {
+        *isVector = true;
+        *isPolar = true;
+        name = MDAL::replace( standardName, "_speed", "_velocity" );
```

Comment:
Why we want to replace speed with velocity? i would keep it as in original data

Reply:
Because the direction name contain velocity and, given the choice, it is preferable to be consistent with other possible variable name.

Reply:
I would keep their names. It is their format and their name choices for any reasons they have...

Reply:
ok. So with direction : velocity_from_direction --> speed